<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qlnhapsach_SuaLoSach</class>
 <widget class="QDialog" name="qlnhapsach_SuaLoSach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../new_resource.qrc">
    <normaloff>:/newPrefix/icons/editLS.png</normaloff>:/newPrefix/icons/editLS.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Bahnschrift</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Mã lô sách:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMaLoSach">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mã sách:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtChonMaSach">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Nhập mã sách hoặc tên sách có trong hệ thống...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mã nhà cung cấp:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbbChonMaNCC">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Chọn nhà cung cấp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-------------------</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Đơn giá:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtDonGia">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Số lượng:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtSoLuong">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Thành tiền:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtThanhTien">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Giá bán lẻ:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtGiaBanLe">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnThoat">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FF6666;
border: 1px solid #FF6666;</string>
   </property>
   <property name="text">
    <string>Hủy</string>
   </property>
   <property name="icon">
    <iconset resource="../new_resource.qrc">
     <normaloff>:/newPrefix/icons/exitLS.png</normaloff>:/newPrefix/icons/exitLS.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSuaLoSach">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #00FF66;
border: 1px solid #00FF66;</string>
   </property>
   <property name="text">
    <string>Sửa lô sách</string>
   </property>
   <property name="icon">
    <iconset resource="../new_resource.qrc">
     <normaloff>:/newPrefix/icons/editLS.png</normaloff>:/newPrefix/icons/editLS.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sửa lô sách</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../new_resource.qrc"/>
 </resources>
 <connections/>
</ui>
